frontend/src/index.tsx

```diff
@@ -26,6 +27,8 @@ export function App() {
             <Route path="/login" component={Login} />
             <Route path="/register" component={Register} />
             <Route path="/learn/:id" component={Learn} />
+            <Route path="/set/:id" component={SetEdit} />
+            <Route path="/setedit" component={SetEdit} />
```

Comment:
edit and view should be same route, just with state for editing a card. Just remove the setedit route